_plugins/version.rb

```diff
@@ -0,0 +1,52 @@
+# Aggregate the En lesson versions together
+Jekyll::Hooks.register :site, :post_read do |site|
```

Comment:
Ok that's super awesome! I believe we could find ways to gameify this (that's a word) so more people can contribute.